packages/notifications/push/amplify_push_notifications_pinpoint/example/lib/main.dart

```diff
@@ -86,6 +65,28 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
+  void bgHandler(PushNotificationMessage pushNotificationMessage) async {
```

Comment:
Now that we are using `engineGroup.createAndRunDefaultEngine(baseContext)` to create and call the default engine, Flutter app's `lib/main.dart` gets called. Which means the Customer's app gets woken up in the background that calls the `Amplify.configure` which in turn configures our event channels and listeners. 

If they want to listen to updates when the app is killed, calling the `Amplify.Notifications.Push.onNotificationReceivedInBackground` in the initState will be good enough. Since the app gets woken up in killed state, this API will get wired up and get fired.